docs/deployment/gae.rst

```diff
@@ -1,225 +1,104 @@
+        - url: /.*
+          script: main.application
 
-   If you do experience this error, you will be able to get around
-   this by zipping libraries. You can use ``pip`` to create zipfiles
-   from packages.  See :ref:`pip_zip` for more information about this.
+    Replace `application` with your App Engine app ID, `version` with the version you want to deploy to, and `pyramid_project`
+    in the `static_dir` definition with the directory name above your static assets. If your static assets are in the root
+    directory, you can just put `static`
 
-   A successful upload looks like so:
+    For more details about app.yaml, see `app.yaml Reference <https://cloud.google.com/appengine/docs/standard/python/config/appref>`_.
 
-   .. code-block:: text
 
-      [chrism@vitaminf pyramidapp]$ python2.5 /usr/local/bin/appcfg.py \
-                                    update ../pyramidapp/app/
-      Scanning files on local disk.
-      Scanned 500 files.
-      # ... more output ...
-      Will check again in 16 seconds.
-      Checking if new version is ready to serve.
-      Closing update: new version is ready to start serving.
-      Uploading index definitions.
+#. Install dependancies
 
-#. Visit ``http://<yourapp>.appspot.com`` in a browser.
+    .. code-block:: text
 
-.. _pip_zip:
+      $ pip install -t lib -r requirements.txt
 
-Zipping Files Via Pip
----------------------
 
-If you hit the Google App Engine 3000-file limit, you may need to
-create zipfile archives out of some distributions installed in your
-application's virtualenv.
+Running Locally
+---------------
 
-First, see which packages are available for zipping:
+    At this point you should have everything you need to run your Pyramid application locally using dev_appserver.
+    Assuming you have appengine in your $PATH,
 
-.. code-block:: text
+    .. code-block:: text
 
-   $ bin/pip zip -l
+      $ dev_appserver.py app.yaml
 
-This shows your zipped packages (by default, none) and your unzipped
-packages. You can zip a package like so:
+    And voilà! You should have a perfectly-running Pyramid application running under Google App Engine, on your local machine.
 
-.. code-block:: text
+Deploying
+---------
 
-   $ bin/pip zip pytz-2009g-py2.5.egg
+    Deploying to App Engine is pretty straight forward. If you've successfully launched your application locally,
+    deploying is just as easy.
 
-Note that it requires the whole egg file name.  For a :app:`Pyramid` app, the
-following packages are good candidates to be zipped.
+    .. code-block:: text
 
-- Chameleon
-- zope.i18n
+      $ appcfg.py update app.yaml
 
-Once the zipping procedure is finished you can try uploading again.
+    Your Pyramid application is now live to the world! You can access it by navigation to your domain name, or by
+    `applicationid.appspot.com`, or if you've specified a version outside of your default, it would
+    be `version-dot-applicationid.appspot.com`
```

Comment:
Dedent flush left.

Optionally remove carriage returns. It's no longer necessary to use them in narrative documentation.